src/client/pages/Country/Country.tsx

```diff
@@ -71,7 +72,8 @@ const Country: React.FC = () => {
 
   if (!countryIso) return null
 
-  if (!Authorizer.canView({ countryIso, assessment, cycle, user })) window.location.href = ClientRoutes.Root.path
+  if (isAppInitialized && !Authorizer.canView({ countryIso, assessment, cycle, user }))
```

Comment:
I think that if App is not initialized we shouldn't even reach the rendering at this point .

Better to return null in PageRoutes if app is not initialized .